edit/provisioning.cattle.io.cluster/rke2.vue

```diff
@@ -333,7 +333,21 @@ export default {
           };
         });
 
-        return sortBy(out, 'sort:desc');
+        const sorted = sortBy(out, 'sort:desc');
+
+        return sorted.filter((version, idx) => {
+          if (semver.prerelease(version.value)) {
+            return true;
+          }
+
+          const nextHighest = sorted[idx - 1];
+
+          if (nextHighest && semver.minor(nextHighest.value) === semver.minor(version.value)) {
+            return false;
+          }
```

Comment:
Shouldn't this actually be checking the major version and not the minor version?

This would filter out 1.21 in:
2.21
1.21

Reading the issue I would think we'd want to keep both.